config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml

```diff
@@ -282,7 +282,7 @@ presubmits:
             - --build=quick
             - --cluster=
             - --env=KUBE_CONTAINER_RUNTIME=containerd
-            - --env=KUBE_UBUNTU_INSTALL_CONTAINERD_VERSION=v1.5.4
+            - --env=KUBE_UBUNTU_INSTALL_CONTAINERD_VERSION=v1.5.5
             - --env=KUBE_UBUNTU_INSTALL_RUNC_VERSION=v1.0.0-rc95
```

Comment:
Should we update runc to 1.0.1 as well?
cc @dims

Reply:
@adisky we use what is recommended with 1.5.5 (https://github.com/containerd/containerd/blob/v1.5.5/script/setup/runc-version) So yay to `1.0.1`